src/ray/core_worker/test/direct_task_transport_test.cc

```diff
@@ -1036,6 +1036,139 @@ TEST(DirectTaskTransportTest, TestKillResolvingTask) {
   ASSERT_EQ(task_finisher->num_tasks_failed, 1);
 }
 
+TEST(DirectTaskTransportTest, TestPipeliningConcurrentWorkerLeases) {
+  rpc::Address address;
+  auto raylet_client = std::make_shared<MockRayletClient>();
+  auto worker_client = std::make_shared<MockWorkerClient>();
+  auto store = std::make_shared<CoreWorkerMemoryStore>();
+  auto factory = [&](const rpc::Address &addr) { return worker_client; };
+  auto task_finisher = std::make_shared<MockTaskFinisher>();
+
+  // setting max_tasks_in_flight_per_worker to a value larger than 1 to enable pipelining
```

Comment:
Can you make sure to use complete sentences for code comments (capitalize and punctuate)?